lua/entities/lunasflightschool_missile.lua

```diff
@@ -188,8 +188,15 @@ if SERVER then
 				effectdata:SetNormal( -self:GetForward() )
 			util.Effect( "manhacksparks", effectdata, true, true )
 
+			local dmgDone = 600
+
+			if HitEnt:IsNPC() or HitEnt:IsPlayer() or HitEnt:IsNextBot() then
+				dmgDone = 100
+
+			end

```

Comment:
I don't think this belongs here - maybe the missile can have a separate method for calculating the damage on a given target. Bonus points if it has a hook

Reply:
Suggestion applied!